### DISCUSSION TITLE
access

### DISCUSSION THREAD

**Discussion by drxx99** [community/community#47317] [Other Feature Feedback, Questions, & Ideas]
need help please

**Comment by ghostinhershell** [comment]
Hi @drxx99, 

Thanks so much for reaching out and posting in GitHub's Community on Discussions. Can you tell us a bit more about what you need help with?

**Comment by cehfisher** [comment]
Hi @drxx99 - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
